test/test-mc-fle2-payload-iev-v2.c

```diff
@@ -83,6 +90,37 @@ static bool _mc_fle2_iev_v2_test_parse(_mc_fle2_iev_v2_test *test, bson_iter_t *
                 TEST_ERROR("Unknown type '%s'", value);
             }
             hasType = true;
+        } else if (!strcmp(field, "edge_count")) {
```

Comment:
This can probably be implicit since we know each edge is precisely 96 bytes.

So:
```
assert((tmp_buf.len % 96) == 0);
test->edge_count = tmp_buf.len / 96;
```

in the "edges" processing down below.